cookbook/core/control_flow/chain_tasks.py

```diff
@@ -1,34 +1,30 @@
 """
-Chain Flyte Tasks
------------------
+Chain Flyte Entities
```

Comment:
@AlekhyaSasi, can you change the file name to chain_entities and also, modify the name in conf.py?

Reply:
Done.